docs/rackhd/install_os.rst

```diff
@@ -69,19 +70,20 @@ All parameters descriptions of OS installation workflow payload are listed below
 rootSshKey          String           *optional*   The public SSH key that will be appended to target OS.
-installDisk         String/Number    *optional*   *installDisk* is to specify the target disk which the OS will be installed on. It can be a string or a number. **For string**, it is a disk path that the OS can recongize, its format varies with OS. For example, "/dev/sda" or "/dev/disk/by-id/scsi-36001636121940cc01df404d80c1e761e" for CentOS/RHEL, "t10.ATA_____SATADOM2DSV_3SE__________________________20130522AA0990120088" or "naa.6001636101840bb01df404d80c2d76fe" or "mpx.vmhba1:C0:T1:L0" or "vml.0000000000766d686261313a313a30" for ESXi. **For number**, it is a RackHD generated disk identifier (it could be obtained from "driveId" catalog). If *installDisk* is omitted, RackHD will assign the default disk by order: SATADOM -> first disk in "driveId" catalog -> "sda" for Linux OS. **NOTE**: Users need to make sure the *installDisk* (either specified by user or by default) is the first bootable drive from BIOS and raid controller setup.
+installDisk         String/Number    *optional*   *installDisk* is to specify the target disk which the OS will be installed on. It can be a string or a number. **For string**, it is a disk path that the OS can recongize, its format varies with OS. For example, "/dev/sda" or "/dev/disk/by-id/scsi-36001636121940cc01df404d80c1e761e" for CentOS/RHEL, "t10.ATA_____SATADOM2DSV_3SE__________________________20130522AA0990120088" or "naa.6001636101840bb01df404d80c2d76fe" or "mpx.vmhba1:C0:T1:L0" or "vml.0000000000766d686261313a313a30" for ESXi. **For number**, it is a RackHD generated disk identifier (it could be obtained from "driveId" catalog). If *installDisk* is omitted, RackHD will assign the default disk by order: SATADOM -> first disk in "driveId" catalog -> "sda" for Linux OS. **NOTE**: Users need to make sure the *installDisk* (either specified by user or by default) is the first bootable drive from BIOS and raid controller setup. **Photon** only supports '/dev/sd*' format currently.
 installPartitions   Array            *optional*   *installPartitions* is to specify the *installDisk*'s partitions when OS installer's default auto partitioning is not wanted. (Only support **CentOS** at present, Other Linux OS will be supported). See installPartitions_ for more details.
-kvm                 Boolean          *optional*   The value is *true* or *false* to indicates to install kvm or not, default is **false**.
+kvm                 Boolean          *optional*   The value is *true* or *false* to indicates to install kvm or not, default is **false**. (**ESXi, CoreOS, Photon** doesn't support this parameter)
```

Comment:
this payload table doesn't include CoreOS, so just mention 'ESXi' and 'Photon' here